core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -900,6 +901,30 @@ private DefaultFunctionResolver year() {
     );
   }
 
+  /**
+   * YEARWEEK(DATE[,mode]). return the week number for date.
```

Comment:
```suggestion
   * YEARWEEK(DATE,[mode]). return the week number for date.
```

Reply:
The syntax is correct as is.  If the mode isn't included, the comma should not be included. This suggestion will say that this is correct syntax:
```
YEARWEEK('12-12-12', )
```